### PR TITLE
feat: add project idea for OpenTelemetry Collector Debug exporter

### PR DESCRIPTION
**Pull request by astencel-sumo**

### CODE REVIEW

File: programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md [hunk 1]
```diff
@@ -122,6 +122,17 @@
...
 
+### OpenTelemetry
+
+#### OpenTelemetry Collector Debug exporter enhancements
+
+- Description: The OpenTelemetry Collector's Debug exporter is not very configurable. Currently the only option is to define `verbosity` at one of three levels: `basic`, `normal`, `detailed`. Users might find it useful to switch certain parts of output on/off, as well as decide where the exporter's output should be sent (currently only `stderr`). Let's add options to the exporter that allow users to configure the exporter's output.
+- Expected Outcome: Debug exporter's output can be configured
+- Recommended Skills: Go
+- Mentor(s):
+  - Andrzej Stencel ([@astencel-sumo](https://github.com/astencel-sumo/), andrzej@andrzejstencel.pl)
```
Comment (jpkrohling) [inline programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md:133]: I can be a co-mentor for this.

**Review by jpkrohling — APPROVED** [review]

**Comment by aliok** [comment]
Thanks for this @astencel-sumo and @jpkrohling 

Everything looks good, but I have a question. I am not an expert in OpenTelemetry but when I checked the upstream ticket, it seemed a bit small to me. In LFX mentorship, we will have mentees working on this project for 12 weeks full time (30-40 hours/week). If you think it is big enough, sure, you're the expert.

I also wanted to let you know that this term we’re doing an information session for mentors. We’d like to invite all the mentors participating to attend one of these if you're able:
* North America: [Jan 29, 2024 10:30 AM (Pacific Time)](https://zoom-lfx.platform.linuxfoundation.org/meeting/99504894798?password=54a6ec2f-d649-432b-a918-acd17720f163)
* EMEA + APAC: [Jan 30, 2024 8:00 AM (UTC)](https://zoom-lfx.platform.linuxfoundation.org/meeting/93514971498?password=907d67b6-3e04-42d3-afdc-52d288122c6f)

**Comment by aliok** [comment]
> I don't really have in mind a bigger solid chunk of work to propose. I could compose a bag of loosely related issues to fix, but I don't think that's a good proposal? Sorry, first time doing this.

If the issues are somewhat related, it can work out.

Or, if there are any possible extensions, that can be better. Like, doing the same work for other modules/components, or, doing some experimentation for integration with other formats/technologies for this issue (I am making things up).

> If this is a problem and there are other well prepared proposals, I'm fine with dropping this proposal in favor of another one that better fits the criteria.

If it is not possible to come up with a larger project, it is fine to drop it. In general, we have flexibility in LFX when it comes to project slots. However, the count is not the issue but in general finding right sized projects is :)

**Comment by nate-double-u** [comment]
What's your thinking, @astencel-sumo and @jpkrohling? Would you like to make updates or shall we postpone till next semester? (We really would love an OTel project in the program 🙂)

**Comment by jpkrohling** [comment]
We have another OTel proposal already, I think we can mature the idea of this one for the next round. We'll definitely have quite a few things to do with the collector as well. What do you think, @astencel-sumo ?

**Comment by aliok** [comment]
Ok, thanks @jpkrohling .

I will close this PR then. Please feel free to open a new PR for the next LFX term.

**Pull request closed without merging**